gamestonk_terminal/stocks/stocks_helper.py

```diff
@@ -407,22 +426,18 @@ def load(
         return [s_ticker, s_start, s_interval, df_stock]
 
 
-def display_candle(s_ticker: str, s_start: datetime, plotly: bool):
-    """Shows candle plot of loaded ticker
+def display_candle(s_ticker: str, df_stock: pd.DataFrame, plotly: bool):
+    """Shows candle plot of loaded ticker. [Source: Yahoo Finance]
```

Comment:
Could be yfinance/iex or alpha vantage....

Reply:
It was actually not like that when I committed this :) There was a yf.download function within this candle function. But now I changed that, so now it makes sense to add the others